src/theme/components/openPositions.module.css

```diff
@@ -44,6 +44,12 @@
   margin: 18px 0px;
   text-align: center;
 }
+.positionContainer {
+  display: block;
+  overflow: hidden;
+  max-width: 1114px;
```

Comment:
out of curiosity, why this exact number?

Reply:
The website styles are an absolute mess. 1114 is the correct width of the button (gleaned from `.position` [here](https://github.com/iron-fish/website/pull/178/files#diff-059fbad4b0aa6cb3f984fbf16e66aa394bee7981c1480e5353e6c64335931b46R71), the element that this new container now contains sits correctly.

At some point I'd like to redo the entire website because of accumulated debt like this.